HT3.py

```diff
@@ -0,0 +1,59 @@
+initial_text = r"""homEwork:
+	tHis iz your homeWork, copy these Text to variable. 
+
+	You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+
+	it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE. 
+
+	last iz TO calculate nuMber OF Whitespace characteRS in this Text. caREFULL, not only Spaces, but ALL whitespaces. I got 87.
+"""
+import re
+# count whitespaces
+whitespace_cnt = re.findall(' ', initial_text) + re.findall('\t', initial_text) + re.findall('\n', initial_text)

```

Comment:
regexp have `\s` mask for any whitespace character, maybe it is better to use it to reduce amount of code and number of cases to handle

Reply:
Changed, but it wasn't in ppt ;)